python/tank/deploy/tank_commands/core_upgrade.py

```diff
@@ -117,12 +120,11 @@ def run_interactive(self, log, args):
         :param log: std python logger
         :param args: command line args
         """
-
         core_version = self._parse_arguments(args)
 
         self._run(log, False, core_version)
 
-    def _run(self, log, suppress_prompts, core_version):
+    def _run(self, log, suppress_prompts, core_constraint_pattern):
```

Comment:
please update docstring. An example of what valid values are for the pattern would also be useful!